src/components/EditableText.tsx

```diff
@@ -52,6 +52,11 @@ export default class EditableText extends Component<Props, InputState> {
             value={value}
             onChange={e => this.handleEdit(e)}
             onBlur={() => this.handleBlur()}
+            onKeyDown={e => {
+              if (e.key === "Enter") {
```

Comment:
I kept pressing enter to save but it wouldn't!

Also, we should handle escape to revert